app/assets/javascripts/components/story/ExpandedStory/ExpandedStoryOwnedBy.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default ExpandedStoryOwnedBy = (props) => {
```

Comment:
You could simplify this line to:
```js
export default ExpandedStoryOwnedBy = ({ users, story, onEdit }) => {
```

So this way you can remove the explicit `return` and declare the component this way:
```js
export default ExpandedStoryOwnedBy = ({ users, story, onEdit }) => (
  <div className="Story__section">
    <div className="Story__section-title">
      {I18n.translate('activerecord.attributes.story.owned_by')}
    </div>
    <select
      value={story._editing.ownedById || ''}
      className="form-control input-sm"
      onChange={(event) => onEdit({ ownedById: event.target.value })}
    >
      <option value=''>
        ----
      </option>
      {
        users.map((user) => (
          <option
            value={user.id}
            key={user.id}
          >
            {user.name}
          </option>
        ))
      }
    </select>
  </div>
);
```